src/LibraryManager.Contracts/IHostInteraction.cs

```diff
@@ -34,11 +35,36 @@ public interface IHostInteraction
         /// <summary>
         /// Writes a file to disk based on the specified <see cref="ILibraryInstallationState"/>.
         /// </summary>
-        /// <param name="path">The full file path</param>
+        /// <param name="filePath">The relative file path to the WorkingDirectory </param>
         /// <param name="content">The content of the file to write.</param>
         /// <param name="state">The desired state of the finished installed library.</param>
         /// <param name="cancellationToken">A token that allows cancellation of the file writing.</param>
         /// <returns><code>True</code> if no issues occured while executing this method; otherwise <code>False</code>.</returns>
```

Comment:
All of these File IO methods will move to another type once I send the PR for extensibility

Reply:
That's why I need a new PR for extensibility even before we start exposing extensibility. Unless you want this PR to grow even bigger ... :)